examples/dataproc-persistent-history-server/terraform/network.tf

```diff
@@ -0,0 +1,4 @@
+resource "google_compute_network" "hadoop-network" {
```

Comment:
Should this and subnet.tf be refactored to use [this cloud foundation toolkit module](https://github.com/terraform-google-modules/terraform-google-network/blob/master/README.md)?

Reply:
Yes, that is preferred.